### PR TITLE
Feat: add autoscaling column to services

### PR DESCRIPTION
**Pull request by joeclayallday**
**Todo**
• Wire up the logic to properly show if a Service is Enabled or Disabled

I'm guessing the logic will be something similar to this

`{user.otpEnabled ? "Enabled" : "Disabled"}`

But `user.otpEnabled` with your own code.

Mockup
1-App

### CODE REVIEW

**Comment by joeclayallday** [comment]
I'm guessing the logic will be something similar to this

`{user.otpEnabled ? "Enabled" : "Disabled"}`

But `user.otpEnabled` with your own code.

**Comment by aguilinger** [comment]
I think this will be sufficient?

**Comment by almathew** [comment]
@joeclayallday happy to review if you want to move this out of draft

File: src/ui/shared/app/services-detail.tsx [hunk 1]
```diff
@@ -37,6 +41,21 @@ const ServiceListRow = ({
...
+    selectEnvironmentById(s, { id: app.environmentId }),
+  );
+  const stack = useSelector((s: AppState) =>
+    selectStackById(s, { id: environment.stackId }),
+  );
+  const autoscaling = useCache<
+    HalEmbedded<{
+      service_sizing_policies: ServiceSizingPolicyResponse[];
+    }>
+  >(fetchServiceSizingPoliciesByServiceId({ service_id: service.id }));
```
Comment (neurosnap) [inline src/ui/shared/app/services-detail.tsx:54]: This should probably be a hook so we can prevent a bunch of optional chaining everywhere.

**Review by neurosnap — CHANGES_REQUESTED** [review]

**Comment by neurosnap** [comment]
Closing in favor of: https://github.com/aptible/app-ui/pull/597

**Pull request closed without merging**